### PR TITLE
Proliferate AbilityService

### PR DESCRIPTION
**Pull request by gdpelican**

### CODE REVIEW

File: lineman/app/components/group_page/group_page_controller.coffee [hunk 1]
```diff
@@ -21,6 +21,6 @@ angular.module('loomioApp').controller 'GroupPageController', ($rootScope, $rout
       user_id: CurrentUser.id).save()
 
   @showDescriptionPlaceholder = ->
-    CurrentUser.isAdminOf(@group) and !@group.description
+    AbilityService.canAdminister(@group) and !@group.description
```
Comment (robguthrie) [inline lineman/app/components/group_page/group_page_controller.coffee]: naming suggestion, can we call this canAdministerGroup .. I'm assuming it does not work for discussions etc..

Comment (gdpelican) [inline lineman/app/components/group_page/group_page_controller.coffee]: Yep sure thing.

**Comment by robguthrie** [comment]
this is super cool. Thanks James. ok to merge?

**Comment by gdpelican** [comment]
Yes!

**Comment by gdpelican** [comment]
Actually hold off on this, I want some more time with it @robguthrie

**Comment by robguthrie** [comment]
close this as it's all in "vote from email"?

**Comment by gdpelican** [comment]
Merged into #2249

**Pull request closed without merging**